app/src/main/res/values/strings.xml

```diff
@@ -228,6 +228,11 @@ and always knows where you are to not miss where to get off the bus.
 	<string name="np_name_sydney" translatable="false">Transport NSW</string>
 	<string name="np_desc_sydney">Australia, New South Wales, Sydney</string>
 
+	<string name="np_region_pt">Portugal</string>
+	<string name="np_name_pt">Portugal</string>
```

Comment:
I would opt for using *Lisbon* as provider name here (if it only provides information for this city?). What's your opinion here @grote?

Reply:
I think this is Navitia's Portugal region. If it only supports Lisbon at the moment, this could be used instead. However, then it needs to be changed when they support other cities.

Reply:
Okay, then we leave it as it is.